src/main/java/com/eprosima/fastdds/idl/parser/typecode/EnumTypeCode.java

```diff
@@ -27,12 +27,19 @@ public EnumTypeCode(
     @Override
     public long maxSerializedSize(
             long current_alignment)
+    {
+        return maxPlainTypeSerializedSize(current_alignment, 8);
+    }
+
+    @Override
+    public long maxPlainTypeSerializedSize(
+            long current_alignment,
+            long align64)
     {
         long initial_alignment = current_alignment;
 
         current_alignment += 4 + TypeCode.cdr_alignment(current_alignment, 4);
```

Comment:
```suggestion
        current_alignment += 4 + TypeCode.cdr_alignment(current_alignment, align64);
```

Reply:
`align64` is only for types bigger than 4 bytes.